deploy/kubernetes/base/kustomization.yaml

```
@@ -0,0 +1,22 @@
+commonLabels:
+  app: keytransparency
```

nit: The [recommended label](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) is... 
```suggestion
  app.kubernetes.io/name: keytransparency
```

---

Thanks